app/views/maps/show.html.erb

```diff
@@ -9,13 +9,14 @@
 <script>
 	Metamaps.currentSection = "map";
 	Metamaps.currentPage = <%= @map.id.to_s %>;
-    Metamaps.Active.Map = <%= @map.to_json.html_safe %>;
-    Metamaps.Mappers = <%= @allmappers.to_json.html_safe %>;
-    Metamaps.Collaborators = <%= @allcollaborators.to_json.html_safe %>;
-    Metamaps.Topics = <%= @alltopics.to_json(user: current_user).html_safe %>;
-    Metamaps.Synapses = <%= @allsynapses.to_json.html_safe %>;
-    Metamaps.Mappings = <%= @allmappings.to_json.html_safe %>;
-    Metamaps.Messages = <%= @allmessages.to_json.html_safe %>;
-    Metamaps.Stars = <%= @allstars.to_json.html_safe %>;
-    Metamaps.Visualize.type = "ForceDirected";
+  Metamaps.ServerData = Metamaps.ServerData || {}
+  Metamaps.ServerData.ActiveMap = <%= @map.to_json.html_safe %>;
+  Metamaps.ServerData.Mappers = <%= @allmappers.to_json.html_safe %>;
+  Metamaps.ServerData.Collaborators = <%= @allcollaborators.to_json.html_safe %>;
+  Metamaps.ServerData.Topics = <%= @alltopics.to_json(user: current_user).html_safe %>;
+  Metamaps.ServerData.Synapses = <%= @allsynapses.to_json.html_safe %>;
+  Metamaps.ServerData.Mappings = <%= @allmappings.to_json.html_safe %>;
+  Metamaps.Messages = <%= @allmessages.to_json.html_safe %>;
```

Comment:
Great question, I had to look at this. 

I guess it's because they aren't backbone models, they're just arrays (like Metamaps.Metacodes)

So I could stick them in ServerData, but I'd just have to pull them back out later. Not sure what's best yet but I think this code is OK for now